### PR TITLE
Add deactivate functionality to dreamblocks

### PR DESCRIPTION
**Pull request by coloursofnoise**
Reuses the same animation as activation.
[Video](https://discordapp.com/channels/403698615446536203/429775320720211968/711491773708632135)

### CODE REVIEW

File: Celeste.Mod.mm/Mod/Entities/ActivateDreamBlocksTrigger.cs [hunk 1]
```diff
@@ -21,6 +23,15 @@ public ActivateDreamBlocksTrigger(EntityData data, Vector2 offset)
                         dreamBlock.ActivateNoRoutine();
                 }
             }
+            else if (deactivate && level.Session.Inventory.DreamDash) {
```
Comment (0x0ade) [inline Celeste.Mod.mm/Mod/Entities/ActivateDreamBlocksTrigger.cs:26]:
```suggestion
            } else if (deactivate && level.Session.Inventory.DreamDash) {
```

**Review by 0x0ade — CHANGES_REQUESTED** [review]
Code styling is off (see comment in the trigger; lack of spacing in DeactivateNoRoutine and Deactivate in DreamBlock itself), but looks good otherwise.

File: Celeste.Mod.mm/Patches/DreamBlock.cs [hunk 2]
```diff
@@ -29,5 +36,61 @@ public patch_DreamBlock(EntityData data, Vector2 offset)
...
+                SurfaceSoundIndex = 11;
+            }
+        }
+
+        public IEnumerator Deactivate() {
+            Level level = SceneAs<Level>();
+            yield return 1f;
+            Input.Rumble(RumbleStrength.Light, RumbleLength.Long);
+            if (shaker == null) {
+                shaker = new Shaker(true, delegate (Vector2 t) {
```
Comment (0x0ade) [inline Celeste.Mod.mm/Patches/DreamBlock.cs:61]:
```suggestion
                shaker = new Shaker(true, t => {
```

**Comment by coloursofnoise** [comment]
Feel free to give feedback on the "fast" animation.

**Review by 0x0ade — APPROVED** [review]

**Pull request merged**